### PR TITLE
Filled required package.json fields

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @AlexanderMoskovkin, @VasilyStrelyaev
PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,21 @@
 {
   "name": "saucelabs-connector",
   "version": "0.0.1",
+  "description": "Helps to connect local machine to the SauceLabs and start remote selenium webdriver on the side",
```
Comment (AlexanderMoskovkin) [inline package.json]: `...and start a browser with the specified page url`
@VasilyStrelyaev please correct this

File: package.json [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+  "author": "Alexander Moskovkin",
+  "contributors": "Georgiy Abbasov",
+  "homepage": "https://github.com/AlexanderMoskovkin/saucelabs-connector/",
+  "bugs": "https://github.com/AlexanderMoskovkin/saucelabs-connector/issues",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/AlexanderMoskovkin/saucelabs-connector.git"
+  },
+  "keywords": [
```
Comment (AlexanderMoskovkin) [inline package.json]: let's remove the `keywords`. It's redundant

File: package.json (hunk 1 above)
Comment (VasilyStrelyaev) [inline package.json]:
```
  "description": "Helps connect the local machine to SauceLabs and starts a browser with the specified page url",
```

**Comment by georgiy-abbasov** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**